test/parallel/test_tensorflow2_keras.py

```diff
@@ -234,44 +248,42 @@ def compute_expected_value(batch_id):
             sum_per_aggregation = 0.0
             for _ in range(backward_passes_per_step):
                 grads_for_batch = 0.0
-                for rank in range(hvd.size()):
-                    grads_for_batch += rank
 
                 # Apply `average_aggregated_gradients`.
                 grads_for_batch /= float(backward_passes_per_step)
 
                 # Averages across workers.
                 sum_per_aggregation += grads_for_batch / float(hvd.size())
 
-            aggregations_completed = math.floor((batch_id + 1) / backward_passes_per_step)
+            aggregations_completed = math.floor(
+                (batch_id + 1) / backward_passes_per_step)
             return aggregations_completed * sum_per_aggregation
 
         @tf.function
-        def apply_gradients_in_tf_function(gradient_updates, model_variables, **kwargs):
+        def apply_gradients_in_tf_function(grads_and_vars, **kwargs):
             # Apply gradient updates in tf.function to reproduce how it is
             # done inside `model.fit()`.
-            hvd_optimizer.apply_gradients(zip(gradient_updates, model_variables), **kwargs)
+            hvd_optimizer.apply_gradients(grads_and_vars, **kwargs)
 
-        gradients = [tf.constant([float(hvd.rank())])]
-        variables = [tf.Variable([0.0])]
+        var = tf.Variable([0.0])
+        variables = [var]
+        def loss(): return (var - var)
```

Comment:
Same here, minor style nit, just split this fn into two lines.

Reply:
updated.